go.mod

```diff
@@ -7,11 +7,14 @@ require (
 	github.com/rabbitmq/amqp091-go v1.5.0
 	github.com/sirupsen/logrus v1.9.0
 	github.com/stretchr/testify v1.8.0
+	github.com/xeipuuv/gojsonschema v1.2.0
```

Comment:
can we move to a different package ?
this was not update from 2019

there is: https://github.com/santhosh-tekuri/jsonschema which has support for draft-2020-12

Reply:
if this should not be handled in this PR, can we have a follow-up issue about it?

Reply:
Given that it is not a drop in replacement we have 2 options:
1. Rework our code to give the same functionality with the other library.
2.  Fork `github.com/xeipuuv/gojsonschema` and update it our selves

I'll add the issue for now.